pulp_rpm/app/serializers.py

```diff
@@ -458,3 +463,47 @@ def validate(self, data):
             new_data.update({'types': final_types})
 
         return new_data
+
+
+class ModulemdSerializer(SingleArtifactContentSerializer):
+    """
+    Modulemd serializer.
+    """
+
+    name = serializers.CharField(
+        help_text=_("Modulemd name."),
+    )
+    stream = serializers.CharField(
+        help_text=_("Stream name."),
+    )
+    version = serializers.CharField(
+        help_text=_("Modulemd version."),
+    )
+    context = serializers.CharField(
+        help_text=_("Modulemd context."),
+    )
+    arch = serializers.CharField(
+        help_text=_("Modulemd architecture."),
+    )
+    artifacts = serializers.CharField(
+        help_text=_("Modulemd artifacts (rpm packages)."),
```

Comment:
I'd remove `rpm packages`, it potentially can be srpms. I think just `Modulemd artifacts` is clear enough.

Reply:
This is a good point.